repository/src/main/java/org/alfresco/repo/action/executer/ScriptActionExecuter.java

```diff
@@ -198,6 +197,7 @@ protected void executeImpl(Action action, NodeRef actionedUponNodeRef)
     /**
      * @see org.alfresco.repo.action.ParameterizedItemAbstractBase#addParameterDefinitions(java.util.List)
      */
+    @Override

```

Comment:
Is this @Override is required for the fix ?

Reply:
Yes. This is not related to the fix in the ticket rather an already existing PMD issue which now appeared in the build